mlflow/utils/databricks_utils.py

```diff
@@ -10,6 +11,30 @@
 
 _logger = logging.getLogger(__name__)
 
+_env_var_prefix = "DATABRICKS_"
+
+
+def _use_env_var_if_exists(env_var, *, if_exists=lambda x: os.environ[x]):
```

Comment:
Introduced this decorator to make it easier to preserve the existing logic for older runtime versions.